cmake/cable.cmake

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/cmake -P
+
+# Cable: CMake Bootstrap Library <https://github.com/ethereum/cable>
+# Copyright 2019 Pawel Bylica.
+# Licensed under the Apache License, Version 2.0.
+
+# The cable command-line tool, version 1.0.0
+#
+# This CMake script allows installing or updating Cable modules.
+# Commands:
+# - list
+# - install
+# - update
+#
+# You can also include it from CMakeLists.txt to add Cable modules to
+# CMAKE_MODULE_PATH.
+
+if(NOT CMAKE_ARGV0)
```

Comment:
Using `CMAKE_SCRIPT_MODE_FILE` here instead (as suggested in https://stackoverflow.com/questions/51427538/cmake-test-if-i-am-in-scripting-mode) might be more readable.